friends/src/App.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+
+import './App.css';
+import axios from 'axios';
+
+
+
+
+class App extends Component {
+constructor(){
+  super();
+  this.state ={
+    friendsData: []
+  };
+}
+
+
+
+  componentDidMount() {
+    axios.get("http://localhost:5000/friends").then(response => {
+      console.log(response);
+      this.setState({friendsData: response.data})
+    })
+    
+      }
+
+  render() {
+    return (
+      <div className="App">
```

Comment:
remember you can always map an array.  might even be a good idea to map into a new "friend" component...

Reply:
Thanks Brian, I got that part down and my friends data is now rendered!